tembo-stacks/src/stacks/specs/analytics.yaml

```diff
@@ -23,21 +22,45 @@ postgres_config:
   - name: pg_stat_statements.track
     value: top
   - name: shared_preload_libraries
-    value: pg_stat_statements,pg_cron
+    value: pg_stat_statements, pg_cron, pg_later, pg_parquet
   - name: track_io_timing
     value: 'on'
   - name: wal_level
     value: logical
+  - name: pglater.host
+    value: postgresql:///postgres?host=/controller/run
 trunk_installs:
+  - name: hydra_columnar
+    version: 1.1.1
   - name: pg_stat_statements
     version: 1.10.0
   - name: pg_partman
     version: 5.0.1
   - name: pg_cron
     version: 1.6.2
+  - name: pg_later
+    version: 0.1.1
   - name: pg_analytics
     version: 0.2.1
+  - name: pg_parquet
+    version: 0.2.0
+  - name: postgres_fdw
+    version: 1.1.0
+  - name: wrappers
+    version: 0.4.2
+  - name: multicorn
```

Comment:
do we still need this now that we aren't including pg_tier (and parquet_s3_fdw)/.

Reply:
I kept them to have access to all the foreign data wrappers which are built on top of the wrappers API (eg. clerk, prometheus etc. fdws which we've built and a number of fdws which supabase has built).

Reply:
woops, I meant just `multicorn` specifically agree on `wrappers`.

Reply:
oh, fair. Making that update too now.